lib/nostrum/struct/channel.ex

```diff
@@ -29,44 +29,28 @@ defmodule Nostrum.Struct.Channel do
   "<#766435015768539156>"
   ```
 
-  ## Channel Cache
+  ## Channel Caching
 
-  The [`ChannelCache`](`Nostrum.Cache.ChannelCache`) module provides functionality for you to retrieve information about any channel that your application can see. It provides two functions: [`get/1`](`Nostrum.Cache.ChannelCache.get/1`) and [`get!/1`](`Nostrum.Cache.ChannelCache.get!/1`).
+  Channels are cached within the guild object they are a member of. To fetch a cached channel you should use the `Nostrum.Cache.ChannelGuildMapping` cache to map a cache to a guild ID.
```

Comment:
We should document here that DM channels are never cached due to the way Discord gateway works here.